ynr/apps/elections/uk/management/commands/uk_create_elections_from_every_election.py

```diff
@@ -11,16 +11,32 @@
 class Command(BaseCommand):
     help = "Create posts and elections from a EveryElection"
 
-    def handle(self, *args, **options):
-
-        # Get all elections from EveryElection
+    def import_approved_elections(self):
+        # Get all approved elections from EveryElection
         ee_importer = EveryElectionImporter()
         ee_importer.build_election_tree()
 
+        for ballot_id, election_dict in ee_importer.ballot_ids.items():
+            parent = ee_importer.get_parent(ballot_id)
+            election_dict.get_or_create_post_election(parent=parent)
+
+    def delete_deleted_elections(self):
+        # Get all deleted elections from EE
+        ee_importer = EveryElectionImporter({"current": 1, "deleted": 1})
```

Comment:
FYI, this was the line that caused me to write this https://github.com/DemocracyClub/EveryElection/issues/421